### DISCUSSION TITLE
How to turn off folders transliteration ?

### DISCUSSION THREAD

**Discussion by victor-ponamariov** [nuxt/content#2048] [Q&A] [answered]
I use Nuxt 3 and its Content module.

When I fetch my /content/ folder structure the .md files are returned as-is, while folders are transliterated into latin.

Nothing special to provide regarding code, but just in case, here is how I fetch it:

```
const { data: navigation } = await useAsyncData('home', async () => {
  const query = queryContent({
    where: {
      _path: { $contains: '/lessons/' }
    }
  })

  return fetchContentNavigation(query)
})
```

And, for example, one of the folders is called "Базовая грамматика". When I output the navigation via console.log, I can see that title is "Bazovaya Grammatika".

I tried googling and even asked ChatGPT, who suggested using `transpile: false` in nuxt config.

But it didn't help.

**Comment by nobkd** [comment]
That happens because of [`slugify`](https://www.npmjs.com/package/slugify). See https://github.com/nuxt/content/issues/1383#issuecomment-1196237686
That should also happen with filenames. For files, you can change it by using the `_path` property in the front matter, as shown in the comment.
I haven't seen an option to change the behavior by default.

**Comment by victor-ponamariov** [comment]
Oh, for files yes, the path is slugified as well. But that happens in URL, not in the navigation.

So I got it. I display the itle property that I get from front matter file properties. But how to do the same with...folders? I can control how to display filenames, since I can put anything in frontmatter properties, but how to deal with folders =\

**Comment by victor-ponamariov** [comment]
The only thing that I've just thought about is to add "group" or "category" property for each file, and group files manually in JS. Like you know, make an associative array

Group Name => array of files

Each file has group name, so I can create this array

**Comment by nobkd** [comment, marked as answer]
For folders, you should be able to use the [`_dir.yml` file (see second code group)](https://content.nuxtjs.org/guide/displaying/navigation#custom-keys) inside the directory to set the title.

```yml
# Базовая грамматика/_dir.yml
title: 'Базовая грамматика'
```
But I don't know why the title field is transliterated by default...

**Reply by victor-ponamariov** [reply]
Thank you!! Will check it out soon, missed the doc section but it looks like exactly what I need!

**Reply by nobkd** [reply]
I'll also try to find that in the code, because I don't think, that the `title` for directories should be transliterated by default.

**Reply by nobkd** [reply]
Okay. It seems, that in [`src/runtime/transformers/path-meta.ts`](https://github.com/nuxt/content/blob/a12fc4a3d55cc1adf5a20fafc855b33bdae8d7d5/src/runtime/transformers/path-meta.ts) it generates the title for a directory by taking the last part of the `_path` (L30, L45), replacing `-` and spaces and converting it to `pascalCase` (L77). But because the `_path` already is `slugified` it results in a transliteration.

I don't see how I could properly change that, sorry.
